jib-core/src/main/java/com/google/cloud/tools/jib/configuration/BuildContext.java

```diff
@@ -423,12 +436,17 @@ private BuildContext(
     this.shutDownExecutorService = shutDownExecutorService;
     this.alwaysCacheBaseImage = alwaysCacheBaseImage;
     this.registryMirrors = registryMirrors;
+    this.enablePlatformTags = enablePlatformTags;
   }
 
   public ImageConfiguration getBaseImageConfiguration() {
     return baseImageConfiguration;
   }
 
+  public boolean isEnablePlatformTags() {
+    return enablePlatformTags;
+  }
```

Comment:
Can we add an assertion for this in BuildContextTest#testBuilder? https://github.com/GoogleContainerTools/jib/blob/0392bdae1933f6e245dc47d5e07f57cb65d90399/jib-core/src/test/java/com/google/cloud/tools/jib/configuration/BuildContextTest.java#L73